x-pack/plugins/apm/public/components/shared/dependencies_table/get_span_metric_columns.tsx

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiToolTip,
+  RIGHT_ALIGNMENT,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { Breakpoints } from '../../../hooks/use_breakpoints';
+import {
+  ChartType,
+  getTimeSeriesColor,
+} from '../charts/helper/get_timeseries_color';
+import { ListMetric } from '../list_metric';
+import { ITableColumn } from '../managed_table';
+import { FETCH_STATUS } from '../../../hooks/use_fetcher';
+import {
+  asMillisecondDuration,
+  asPercent,
+  asTransactionRate,
+} from '../../../../common/utils/formatters';
+import { Coordinate } from '../../../../typings/timeseries';
+import { ImpactBar } from '../impact_bar';
+import { isFiniteNumber } from '../../../../common/utils/is_finite_number';
+
+export interface SpanMetricGroup {
+  latency: number | null;
+  throughput: number | null;
+  failureRate: number | null;
+  impact: number | null;
+  currentStats:
+    | {
+        latency: Coordinate[];
+        throughput: Coordinate[];
+        failureRate: Coordinate[];
+      }
+    | undefined;
+  previousStats:
+    | {
+        latency: Coordinate[];
+        throughput: Coordinate[];
+        failureRate: Coordinate[];
+        impact: number;
+      }
+    | undefined;
+}
+
+export function getSpanMetricColumns({
+  breakpoints,
+  comparisonFetchStatus,
+}: {
+  breakpoints: Breakpoints;
+  comparisonFetchStatus: FETCH_STATUS;
+}): Array<ITableColumn<SpanMetricGroup>> {
+  const { isLarge } = breakpoints;
+  const shouldShowSparkPlots = !isLarge;
```

Comment:
Do we need to change this? (used to be `isXl`)

Reply:
`isSingleColumn` was true when `isLarge` is true, so I've simplified it.